src/main/resources/db/migration/0200/V249__ProjectVoting.sql

```diff
@@ -0,0 +1,27 @@
+-- Add vote options enum table (YES, CONDITIONAL, NO)
+CREATE TABLE accelerator.vote_options (
+    id INTEGER PRIMARY KEY GENERATED BY DEFAULT AS IDENTITY,
+    value TEXT NOT NULL,
+    UNIQUE (value)
+);
+
+-- Add whitelisted voters per project
+CREATE TABLE accelerator.project_voters (
+    user_id BIGINT NOT NULL REFERENCES users ON DELETE CASCADE,
+    project_id BIGINT NOT NULL REFERENCES projects ON DELETE CASCADE,
+    PRIMARY KEY (user_id, project_id)
+);
+
+-- Add voting records
+CREATE TABLE accelerator.project_vote_records (
+    user_id BIGINT NOT NULL REFERENCES users ON DELETE CASCADE,
+    project_id BIGINT NOT NULL REFERENCES projects ON DELETE CASCADE,
```

Comment:
I know you're discussing this with the product folks, but I think this is going to need a phase ID too and the primary key will be (user, project, phase). I believe there's a vote at each phase transition, so a project will be voted on more than once over its lifetime.